test/Test/Database/LSMTree/Internal/Entry.hs

```diff
@@ -6,16 +6,12 @@
 
 module Test.Database.LSMTree.Internal.Entry (tests) where
 
-import           Data.Coerce
 import           Data.List.NonEmpty (NonEmpty)
 import           Data.Semigroup hiding (First)
 import qualified Data.Semigroup as S
-import           Data.Void
-import qualified Database.LSMTree as Unified
+import qualified Database.LSMTree as Update
```

Comment:
Also, let's call it `Full` here

Reply:
Also done in 367e935f1bc2d8be6ad01bcc4ae92a178d771816.